src/components/cards/NavMenuCard/NavMenuCard.tsx

```diff
@@ -371,36 +377,48 @@ function NavMenuCard() {
         <MenuButton
           setShow={setShowMenu}
           show={showMenu}
-          className={walletAddress ? 'outline-button' : ''}
+          className={
+            walletAddress
+              ? 'outline-button items-center justify-center'
+              : 'items-center justify-center'
+          }
           style={
             walletAddress
               ? {
                   borderRadius: 'var(--corner-radius',
                   borderWidth: '1px',
-                  padding: '10px',
+                  padding: '0px',
                   borderColor: 'var(--text-faded)',
                   width: 'fit-content',
                   minWidth: 'unset',
                 }
               : {}
           }
         >
-          {primaryNameData?.name ? (
-            <span className="flex text-white fill-success gap-2 items-center text-[14px]">
-              <svg width="8px" height="8px">
-                <circle cx="4" cy="4" r="4" fill="current" />
-              </svg>
-              {shortPrimaryName(primaryNameData.name)}
-            </span>
-          ) : walletAddress ? (
-            <WalletAddress characterCount={4} />
-          ) : (
-            <MenuIcon
-              width={'24px'}
-              height={'24px'}
-              fill={'var(--text-white)'}
-            />
-          )}
+          <span className="flex text-white gap-2 items-center justify-center text-[14px] h-[2.5rem] p-2">
+            {primaryNameData?.name ? (
+              <>
+                <AntLogoIcon
+                  className="rounded-full max-h-[1.875rem] border border-dark-grey"
+                  id={domainDomain?.info?.Logo}
+                  icon={
+                    <div className="flex items-center justify-center rounded-full fill-white p-[5px]">
```

Comment:
In general need to stop using px and use rem by default. This will need to get fixed during audit.